testutil/manual_blobtx_gen.go

```diff
@@ -0,0 +1,177 @@
+package testutil
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/testutil/blobfactory"
+	blobtypes "github.com/celestiaorg/celestia-app/x/blob/types"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	coretypes "github.com/tendermint/tendermint/types"
+)
+
+// RandBlobTxsWithAccounts will create random blob transactions using the
+// provided configuration. The account info is queried directly from the
+// application. One blob transaction is generated per account provided.
+func RandBlobTxsWithAccounts(
```

Comment:
adding to our testing technical debt a bit by continuing to copy paste minor alterations of this function. #1114 

we have to keep this in the testutil pkg instead of blob factory because we're importing the app to query directly, and don't want to cause an import cycle in blobfactory